server/yarn.lock

```diff
@@ -0,0 +1,3750 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Please add this file to .gitignore and remove it from the branch.
This file contains sub-dependencies and should be updated automatically by package.json.